dre_probe_matcher.py

```
@@ -0,0 +1,649 @@
+                })
+        print(f"Found {found} out of {total} probes")
+        st_align = {}
+        if RUN_ALIGNMENT:
+            try:
+                targets = ['vol-human-8022671-SplitHighMulti', 'qol-human-2932740-HighExtreme', 'vol-human-1774519-SplitHighMulti', 'qol-human-6349649-SplitHighMulti', 
+                        'vol-human-6403274-SplitEvenBinary', 'qol-human-3447902-SplitHighMulti', 'vol-human-7040555-SplitEvenBinary', 'qol-human-7040555-SplitHighMulti', 
+                        'vol-human-2637411-SplitEvenMulti', 'qol-human-3043871-SplitHighBinary', 'vol-human-2932740-SplitEvenMulti', 'qol-human-6403274-SplitHighBinary', 
+                        'vol-human-8478698-SplitLowMulti', 'qol-human-1774519-SplitEvenBinary', 'vol-human-3043871-SplitLowMulti', 'qol-human-9157688-SplitEvenBinary', 
+                        'vol-human-5032922-SplitLowMulti', 'qol-human-0000001-SplitEvenMulti', 'vol-synth-LowExtreme', 'qol-human-8022671-SplitLowMulti', 'vol-synth-HighExtreme', 
+                        'qol-human-5032922-SplitLowMulti', 'vol-synth-HighCluster', 'qol-synth-LowExtreme', 'vol-synth-LowCluster', 'qol-synth-HighExtreme', 'vol-synth-SplitLowBinary', 
+                        'qol-synth-HighCluster', 'qol-synth-LowCluster', 'qol-synth-SplitLowBinary']
+                for target in targets:
+                    if ('vol' in target and 'vol' not in self.soartech_yaml['id']) or ('qol' in target and 'qol' not in self.soartech_yaml['id']):
+                        continue
+                    st_align[target] = self.get_session_alignment(f'{ST_URL}/api/v1/response', 
+                                            f'{ST_URL}/api/v1/alignment/session?session_id={self.soartech_sid}&target_id={target}',
+                                            match_data,
+                                            self.soartech_sid,
+                                            self.soartech_yaml['id'])
+            except:
+                pass
```

Are you sure we should silently ignore failures to get session alignment from TA1?  If `RUN_ALIGNMENT` is True, won't we want to know if it wasn't obtained?

---

Good point, I added a log here